src/Pagerfanta/View/Template/Template.php

```diff
@@ -26,11 +26,17 @@ public function __construct()
         $this->initializeOptions();
     }
 
+    /**
+     * @param $routeGenerator
```

Comment:
this phpdoc *could* be useful if it was documenting the type. Given it does not, it is totally useless (and invalid)